src/Aquifer.API/Endpoints/Resources/Content/PullFromPublisherReview/Request.cs

```diff
@@ -0,0 +1,9 @@
+namespace Aquifer.API.Endpoints.Resources.Content.PullFromPublisherReview;
+
+public record Request
+{
+    public int? ContentId { get; set; }
+    public int[]? ContentIds { get; set; }
+    public int AssignedUserId { get; set; }
+    public int? AssignedReviewerUserId { get; set; }
```

Comment:
Again, I don't think this should be here. You don't assign a reviewer this way.

Reply:
forgot